addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -468,11 +468,14 @@ ListRenderer.include({
      * prevent subsequent editions. These edits would be lost, because the list
      * view only saves records when unselecting a row.
      *
+     * @param {Object} [options]
+     * @param {boolean} [options.discardIfNotDirty=false] set to true when
+     *   user clicks anywhere on the screen except save button.
```

Comment:
this docstring says when/how the option is used. This is not what a docstring should be: it should rather explain what the option does.